newrelic-jaxrs/pom.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <artifactId>beadledom-parent</artifactId>
+    <groupId>com.cerner.beadledom</groupId>
+    <version>2.8-SNAPSHOT</version>
```

Comment:
Should this part of the 3.x release?

Reply:
Yup, it will also be merged into the 3.x branch.

Reply:
For additional clarification, since it's a new module it can be part of 2.x.